dockerfiles/base/scripts/base/startup.sh

```diff
@@ -251,20 +255,40 @@ init() {
   source "${SCRIPTS_CONTAINER_SOURCE_DIR}"/cli.sh
 }
 
+cleanup() {
+  RETURN_CODE=$?
+
+  # CLI developers should only return '3' in code after the init() method has completed.
+  # This will check to see if the CLI directory is not mounted and only offer the error
+  # message if it isn't currently mounted.
+  if [ $RETURN_CODE -eq "3" ]; then
```

Comment:
As far as I understand we should check that no code is using exit or return with exit code that differs from 3 in that case. Am I right? Have you grepped CLI sources such a code?